### DISCUSSION TITLE
Bisq linux repos

### DISCUSSION THREAD

**Discussion by alkum** [bisq-network/bisq#6118] [General]
Every Bisq release is packaged in different formats. For linux, we use `deb` and `rpm`. These are provided as release artefacts, so linux users have to manually download, verify and install them on every release.

One way to streamline this is to publish the linux binaries in `deb` and `rpm` linux repos (see #4563).

I looked into this and found that it's quite simple to create a `deb` or `rpm` repo. All that's needed is a webserver to serve static content[^1]. The bisq binary would then be copied there and repo metadata generated.

As a test, I tried to publish such a repo using GitHub Pages. However, our release binary is >100MB so it needs to be pushed via Git LFS. Unfortunately, GitHub Pages doesn't support Git LFS (see https://github.com/git-lfs/git-lfs/issues/791).

Could this be done from another Bisq server, which is under our (i.e. a bonded role's) control? Like a dedicated subdomain, for example https://deb.bisq.network (cc https://github.com/bisq-network/roles/issues/18)?

Or would it be better to try to host this on a 3rd party [^2]?

[^1]: A static webserver, 3-400 MB of space and some way to new binaries (ssh, ftp, etc) during a release.
[^2]: https://www.jfrog.com/confluence/display/JFROG/Debian+Repositories

**Comment by ripcurlx** [comment]
I think it would be a good idea to make the update/install process for Linux users more user friendly.
